apps/showcase/package.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "@rnr/showcase",
+  "main": "index.js",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "expo start -c",
+    "android": "expo start -c --android",
+    "dev": "expo start -c --ios",
+    "dev:web": "expo start -c --web",
+    "clean": "rm -rf .expo node_modules"
+  },
+  "dependencies": {
+    "@rnr/reusables": "*",
+    "@rnr/accordion": "*",
+    "@rnr/alert-dialog": "*",
+    "@rnr/aspect-ratio": "*",
+    "@rnr/avatar": "*",
+    "@rnr/checkbox": "*",
+    "@rnr/collapsible": "*",
+    "@rnr/context-menu": "*",
+    "@rnr/dialog": "*",
+    "@rnr/dropdown-menu": "*",
+    "@rnr/hover-card": "*",
+    "@rnr/label": "*",
+    "@rnr/menubar": "*",
+    "@rnr/navigation-menu": "*",
+    "@rnr/popover": "*",
+    "@rnr/portal": "*",
+    "@rnr/progress": "*",
+    "@rnr/radio-group": "*",
+    "@rnr/select": "*",
+    "@rnr/separator": "*",
+    "@rnr/slider": "*",
+    "@rnr/slot": "*",
+    "@rnr/switch": "*",
+    "@rnr/table": "*",
+    "@rnr/tabs": "*",
+    "@rnr/toast": "*",
+    "@rnr/toggle": "*",
+    "@rnr/toggle-group": "*",
+    "@rnr/toolbar": "*",
+    "@rnr/tooltip": "*",
+    "@rnr/types": "*",
+    "@gorhom/bottom-sheet": "^4.6.0",
+    "@hookform/resolvers": "^3.3.4",
+    "@radix-ui/react-accordion": "^1.1.2",
```

Comment:
You can remove the radix-ui dependencies in this area. They are no longer available from here because mono has added packages in the repo. @mrzachnugent